be/src/exec/broker_reader.cpp

```diff
@@ -155,6 +155,8 @@ Status BrokerReader::readat(int64_t position, int64_t nbytes, int64_t* bytes_rea
             return status;
         }
 
+        LOG(DEBUG) << "send readat request to broker:" << broker_addr << " position:" << position << ", read bytes length:" << nbytes;
```

Comment:
what is "readat ?"

Reply:
> what is "readat ?"

Like `pread`

Reply:
Yes, it's confusing.  Better named it as pread request.